reportTemplatesSrc/components/HeaderTextbox.js

```diff
@@ -0,0 +1,16 @@
+const React = require('react');
+const headerTextboxStyle = require('../styles/headerTextbox');
+
+module.exports = ({ data, title}) => {
+  return (
+    <div className='header-text-container'>
+      <div className='header-title'>
+        {title}
+      </div>
+      <div className='text-item'>
+        {data?data:'Not indicated'}
```

Comment:
For readability, put space between items in a ternary expression (some of the projects have prettier that does this for you): `{ a ? b : 'c'}`